package.json

```diff
@@ -14,7 +14,7 @@
     "lint": "eslint . --report-unused-disable-directives",
     "flow": "flow --include-warnings",
     "test": "node node_modules/jest/bin/jest.js --config=.jest.json",
-    "test-ci": "yarn flow && yarn lint && yarn test --ci --color --maxWorkers=2",
+    "test-ci": "yarn flow && yarn lint && yarn test --ci --color --maxWorkers=3",
```

Comment:
I am not sure why this would solve anything: --maxWorkers option only applies to `yarn test`, and the issue in CI is spawning the flow server.